src/pocketmine/Player.php

```diff
@@ -2309,7 +2309,7 @@ public function handleDataPacket(DataPacket $packet){
 					break;
 				}
 
-				if($packet->protocol1 !== ProtocolInfo::CURRENT_PROTOCOL){
+				if(!in_array($packet->protocol1, ProtocolInfo::ACCEPTED_PROTOCOLS)){
 					if($packet->protocol1 < ProtocolInfo::CURRENT_PROTOCOL){
```

Comment:
Maybe you should change this line to `if($packet->protocol1 < min(ProtocolInfo::SUPPORTED_PROTOCOLS))` and similar for the other end.